k8s/base/crc

```diff
@@ -0,0 +1 @@
+ADMIN_PASSWORD=pass
```

Comment:
Shouldn't the username also be in this secret?

Reply:
Well, the username defaults to `admin`, so I figured it didn't need to be set here. I'm happy to add it if you prefer it to be explicit.

Reply:
I do prefer to be explicit especially dealing with other developers